CHANGELOG.md

```diff
@@ -15,7 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed a bug that would make the VM to loop and failing to process selected fds on Linux
 - Fixed classes of exceptions in estdlib.
 - Fixed STM32 code that was hard coded to the default target device, now configured based on the `cmake -DDEVICE=` parameter
-
+- Fixed hard fault on STM32 durung malloc on boards with more than one bank of sram
+ 
```

Comment:
Here the diff looks odd, can you double-check if we added any white space error by mistake?

Reply:
No errors in the white space, the blank line; (formerly line 18) between
`- Fixed STM32 code that was hard coded to the default target device...` (line 17)
and the `### Changed` section (formerly line 19), has moved down to line 19 to make room for the new entry, and the `### Changed` section now starts on line 20.